microservices/scraper-ngoindia/app/routes.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask
+from scraper import get_page_data
+
+
+app = Flask(__name__)
+
+@app.route('/')
+def my_first_route():
+	return "<h1> Hello World! </h1>"
+
+@app.route('/data')
+def page_data():
+	return str(get_page_data())
+
```

Comment:
You can rename the my_first_route() to something more specific and maybe have it return all the data that's scraped, but that's up to you. Also you should have an endpoint that returns a list of all the endpoints that are available to use for the CLI. The code should be the same as what's in the other scrapers, so you can also look off of those for reference.